packages/compiler/core/checker.ts

```diff
@@ -4674,33 +4671,54 @@ export function filterModelProperties(
   return finishTypeForProgram(program, newModel);
 }
 
+/**
+ * Gets the property from the nearest base type that is overridden by the
+ * given property, if any.
+ */
```

Comment:
I figured this would be generally useful if you really need to get overriden properties so I documented it and exported it. I think you had something almost identical in an earlier revision.